tests/unit/samplegen/test_manifest.py

```diff
@@ -75,17 +76,18 @@ def test_generate_manifest():
                                           "region_tag", "giant_clam_sample")
                                   ],
                               ])
-    ]
+    ])
 
-    assert info == expected_info
+    assert info == doc
 
     expected_rendering = dedent("""
+                                ---
```

Comment:
I would omit the `region_tag` below when empty

Reply:
Punting to a separate issue: #172